.devcontainer/boot.sh

```diff
@@ -0,0 +1,9 @@
+sudo chown -R vscode:vscode ${BUNDLE_PATH} && \
+    bundle install --jobs=3 --retry=3 && \
+    bundle exec appraisal install
+
+# Rebuild PostgreSQL
+dropdb --if-exists devise_security_test
```

Comment:
Can you use the rails commands here instead (not sure)?

Reply:
We can but it will add more complexity as `boot.sh` uses PostgreSQL builtin CLI to rebuild the database regardless it is DevContainer/Codespaces or host development.

Reply:
Ok, this is fine then